DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -84,19 +84,13 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('cache_manager.xml');
         }
 
-        $container->setParameter($this->getAlias().'.compiler_pass.tag_annotations', $config['tags']['enabled']);
         if ($config['tags']['enabled']) {
```

Comment:
you are right, this is always defined and the Configuration class pre-validation makes sure that proxy_client is defined when tags are enabled (as its disabled if there is no cache_manager, and there can be no cache_manager if there is no proxy_client)